orderer/common/follower/follower_chain.go

```diff
@@ -374,12 +386,15 @@ func (c *Chain) pullUpToJoin() error {
 	return nil
 }
 
-// pullAfterJoin pulls blocks continuously, inspecting the fetched config blocks for membership.
-// On every config block, it renews the BlockPuller, to take in the new configuration.
-// It will exit with 'nil' if it detects a config block that indicates the orderer is a member of the cluster.
-// It checks whether the chain was stopped between blocks.
+// pullAfterJoin pulls blocks continuously, inspecting the fetched config
+// blocks for membership. On every config block, it renews the BlockPuller,
+// to take in the new configuration. It will exit with 'nil' if it detects
+// a config block that indicates the orderer is a member of the cluster. It
+// checks whether the chain was stopped between blocks.
 func (c *Chain) pullAfterJoin() error {
 	c.setStatus(types.StatusActive)
+	clusterRelation, status := c.StatusReport()
+	c.clusterMetricsReporter.ReportClusterRelationAndStatusMetrics(c.ledgerResources.ChannelID(), clusterRelation, status)
```

Comment:
Why not embed this into `setStatus`?  You need to call it any time the status is updated, and, you won't have to re-acquire the lock that's done in StatusReport().  Also, if someone ever sets status elsewhere, they won't have to remember to update the metric.

Reply:
Good idea. Done (also added to setClusterRelation).